### DISCUSSION TITLE
Give a more proper title instead of `in this module`

### DISCUSSION THREAD

**Discussion by awxiaoxian2020** [mdn/mdn-community#265] [Content] [answered]
See https://github.com/mdn/content/pull/21625

**Comment by wbamberg** [comment]
The Learn area content has a pretty well-defined three-level structure.

- at the top level there are **courses** (e.g. https://developer.mozilla.org/en-US/docs/Learn/JavaScript)
- each course is a series of **modules** (e.g. https://developer.mozilla.org/en-US/docs/Learn/JavaScript#modules, https://developer.mozilla.org/en-US/docs/Learn/CSS#modules, ...) 
- each module is a series of **guides** (e.g. https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps#guides), which more or less correspond to individual pages

Modules are ordered within a course (so elementary stuff comes before advanced) and guides are ordered within a module.

So "In this module" on a guide page provides navigation across all the other guide pages in the same module. It lists the guides in the order they are presented and helps a user see where they are in the module, as well as navigating to other guide pages.

In that light "In this module" seems like a reasonable title to me. Why does it seem wrong to you?

Regarding the PR you linked: I don't think "See also" is the same: this is used on MDN for rather loosely-related pages that may be relevant to the current one, and are never presented in a particular order.

I do think we could do better at expressing the Learn area's structure, but it would be a project.

**Reply by awxiaoxian2020** [reply]
Maybe it's a difference in language habits, I think it's weird for a prepositional phrase to be a title.

@yin1999 What do you think about it?

**Reply by yin1999** [reply]
Agree with Will. In learn area, the structure maybe a bit same with some learning books.

**Reply by wbamberg** [reply]
> Maybe it's a difference in language habits, I think it's weird for a prepositional phrase to be a title.
> 

Then "Module contents" or "Articles in this module" could be better? (tbh I'm not sure what is a good name for the components of a module).

**Reply by awxiaoxian2020** [reply]
> > Maybe it's a difference in language habits, I think it's weird for a prepositional phrase to be a title.
> 
> Then "Module contents" or "Articles in this module" could be better? (tbh I'm not sure what is a good name for the components of a module).

I think so.

**Reply by wbamberg** [reply]
I asked around a bit among the reviewers and feedback was: "Articles in this module" is better than what we have now, but we don't feel very strongly about it. Meaning, we'd happily accept a PR to make this change, as long as:
- it doesn't include other changes (i.e. is easy to review)
- it is complete (we really don't want half to say one thing and half another)

But, I think we should leave this discussion open for a few more days. I didn't get a lot of feedback yet.

**Comment by awxiaoxian2020** [comment]
@wbamberg 
One more thing, is there an article telling readers about the structure of MDN? Although there are [roadmap articles](https://developer.mozilla.org/en-US/docs/Learn/Front-end_web_developer), I think it might be better to tell the reader what each section does and what is there, and let them choose for themselves.

**Reply by wbamberg** [reply]
Do you mean MDN as a whole or the Learn area in particular? In either case the answer would be "no" I suppose.

I think it would be a good idea to have something (on https://developer.mozilla.org/en-US/docs/Learn, maybe), explaining the structure of Learn.

**Comment by estelle** [comment]
Maybe "Lessons in this module" "Guides in this module" and "articles in this module" depending on if they are lessons, guides, or articles.

**Comment by awxiaoxian2020** [comment, marked as answer]
Tracked in https://github.com/mdn/content/issues/24467
